Updated Hal Finkel DOE video link in funder series (#1011)
USRSE/usrse.github.io: _events/2022/2022-10-funder-talk-series.md

## Event

> The inaugural talk of the US-RSE Funder Talk Series will be with Hal Finkel, Ph.D. from the US Department of Energy's (DOE's) Office of Advanced Scientific Computing Research (ASCR). This event took place **Monday, October 17th at 2 PM ET, 1 PM CT, 12 PM MT, 11 AM PT**

Reviewer: ```suggestion
The inaugural talk of the US-RSE Funder Talk Series featured Hal Finkel, Ph.D. from the US Department of Energy's (DOE's) Office of Advanced Scientific Computing Research (ASCR). This event took place **Monday, October 17th at 2 PM ET, 1 PM CT, 12 PM MT, 11 AM PT**
```

Reviewer: Just one language thing - otherwise, fabulous!

Author: Committed. No idea why this is still listed as 'change suggested' below. I checked before the last request and it was changed to featured.
